README.md

```diff
@@ -5,16 +5,24 @@ Application for demonstrating surface blending with ros
 
 Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C3%B6del
 
-
 ### Installation
 ==============
```

Comment:
Just noticed this: it's unnecessary to have both `###` as prefix, and `======` underneath your headers in markdown (in fact, it results in strange rendering of your document, with double spacings, etc).